utils/ExportUtils.js

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {XH} from '@xh/hoist/core';
+import {ToastManager} from '@xh/hoist/toast';
+import {fmtDate} from '@xh/hoist/format';
+import {orderBy, uniq, isString, isFunction} from 'lodash';
+import download from 'downloadjs';
+
+
+/**
+ * Cell formats used in excel exports.
+ * Can be applied to columns via the 'exportFormat' prop.
+ */
+export const exportFormats = {
```

Comment:
Seems like our naming convention would have this capitalized and singular -- e.g. ExportFormat